### 1 - Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800]. Найдите ее среднее значение и дисперсию.

In [93]:
def mean_and_variance(a,b):
    return f'На промежутке ({a};{b}]\nСреднее значение: {(a+b)/2: .2f}\nДисперсия: {((b-a)**2)/12: .2f}'
print(mean_and_variance(200,800))

На промежутке (200;800]
Среднее значение:  500.00
Дисперсия:  30000.00


### Ответ:
#### Среднее значение - ***500.00***
#### Дисперсия - ***30000.00***

### 2 - О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2. Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5? Если да, найдите ее.

In [94]:
b=0.5+2.4**(1/2)
print(f'Правая граница {b: .3f}\n'
      f'Среднее значение{(b+0.5)/2: .3f}')

Правая граница  2.049
Среднее значение 1.275


### Ответ:
#### ***Правая граница  2.049***
#### ***Среднее значение 1.275***

### 3 - Непрерывная случайная величина X распределена нормально и задана плотностью распределения f(x) = (1 / (4 * sqrt(2pi))) * exp((-(x+2)**2) / 32)
#### Найдите:
#### а) M(X)
#### б) D(X)
#### в) std(X) (среднее квадратичное отклонение)

### Ответ:
#### а) ***M(Х) = -2***
#### б) ***D(Х) = 16***
#### в) ***std(Х) = 4***

### 4 - Рост взрослого населения города X имеет нормальное распределение. Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см. Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:

#### а) больше 182 см
#### б) больше 190 см
#### в) от 166 см до 190 см
#### г) от 166 см до 182 см
#### д) от 158 см до 190 см
#### е) не выше 150 см или не ниже 190 см
#### ё) не выше 150 см или не ниже 198 см
#### ж) ниже 166 см.

In [95]:
from statistics import NormalDist

def z_value(height):
    return (height-174)/8 # функция для определний Z-значений

## а)

In [96]:
print(z_value(182))

1.0


In [97]:
NormalDist().cdf(z_value(182))

0.8413447460685428

In [98]:
P=1-NormalDist().cdf(z_value(182))
P

0.15865525393145719

## б)

In [99]:
z=z_value(190)
z

2.0

In [100]:
P=1-NormalDist().cdf(z)
P

0.022750131948179098

## в)

In [101]:
z1=z_value(166)
z1

-1.0

In [102]:
z2=z_value(190)
z2

2.0

In [103]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
P

0.8185946141203637

## г)

In [104]:
z1=z_value(166)
z1

-1.0

In [105]:
z2=z_value(182)
z2

1.0

In [106]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
P

0.6826894921370856

## д)

In [107]:
z1=z_value(158)
z1

-2.0

In [108]:
z2=z_value(190)
z2

2.0

In [109]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
P

0.9544997361036418

## е)

In [110]:
z1=z_value(150)
z1

-3.0

In [111]:
z2=z_value(190)
z2

2.0

In [112]:
P=NormalDist().cdf(z1)+(1-NormalDist().cdf(z2))
P

0.0241000299798092

## ё)

In [113]:
z1=z_value(150)
z1

-3.0

In [114]:
z2=z_value(198)
z2

3.0

In [115]:
P=NormalDist().cdf(z1)+(1-NormalDist().cdf(z2))
P

0.002699796063260207

## ж)

In [116]:
z=z_value(166)
z

-1.0

In [117]:
P=NormalDist().cdf(z)
P

0.15865525393145713

### Ответ:
#### a) ***0.15865525393145719***;
#### б) ***0.022750131948179098***;
#### в) ***0.8185946141203637***;
#### г) ***0.6826894921370856***;
#### д) ***0.9544997361036418***;
#### е) ***0.0241000299798092***;
#### ё) ***0.002699796063260207***;
#### ж) ***0.15865525393145713***;

### 5 - На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

In [118]:
Z=(190-178)/25**(1/2)
Z

2.4

### Ответ:
#### на ***2,4*** сигмы